numpy/tests/typing/pass/array_like.py

```diff
@@ -1,13 +1,7 @@
 from typing import Any, List, Optional, TYPE_CHECKING
 
 import numpy as np
-
-if TYPE_CHECKING:
-    from numpy.typing import ArrayLike, DtypeLike, _SupportsArray
-else:
-    ArrayLike = Any
-    DtypeLike = Any
-    _SupportsArray = Any
+from numpy.typing import ArrayLike, DtypeLike, _SupportsArray
```

Comment:
We also execute all the code in `pass`, so we're also testing here that you can really import these things at runtime.